base/setup/lib/utils/partlist.h

```diff
@@ -34,10 +34,32 @@ typedef enum _FORMATSTATE
     Unformatted,
     UnformattedOrDamaged,
     UnknownFormat,
-    Preformatted,
     Formatted
 } FORMATSTATE, *PFORMATSTATE;
 
+#include "volutil.h"
+
+typedef struct _PARTENTRY PARTENTRY, *PPARTENTRY;
+typedef struct _VOLENTRY
+{
+    LIST_ENTRY ListEntry; //< Entry in VolumesList or PendingUnmountVolumesList
+
+    VOLINFO Info;
+    FORMATSTATE FormatState;
+
+/** The following properties may be replaced by flags **/
```

Comment:
Can you clarify this statement?

Reply:
I will remove it because it's misleading and doesn't bring much of information (Mark also asked me about it). Basically this is a "maybe-consider-todo-in-the-future" type of comment.